test/optimizer/zonemaps.test

```diff
@@ -0,0 +1,85 @@
+# name: test/optimizer/zonemaps.test
+# description: Test if zonemaps are correctly used
+# group: [optimizer]
+
+statement ok
+PRAGMA explain_output = PHYSICAL_ONLY;
+
+statement ok
+create temporary table t as select range a, length(range) b, mod(range,10000) c, 5 d, 10000 e from range(100000);
+
+# Zonemap checks
+
+query II
+explain select count(*) from t where a >=2 and a < 7 or a in (1,3,8);
```

Comment:
I would like to see more tests, including tests with different combinations of bounds (e.g. `a >= 500, a < 700`, `a >= 500, a <= 700`, etc) as well as different orders (`500 <= a`) and different types (tinyint, smallint, integer, bigint - at least one test with each type).